pkg/operator/bootstrap.go

```diff
@@ -61,6 +66,28 @@ func (c *Controller) Bootstrap() error {
 		mgmtState = operatorapi.Removed
 	}
 
+	infra, err := util.GetInfrastructure(c.listers)
+	if err != nil {
+		return err
+	}
+
+	rolloutStrategy := appsapi.RollingUpdateDeploymentStrategyType
+
+	// If Swift service is not available for OpenStack, we have to start using
+	// Cinder with RWO PVC backend. It means that we need to create an RWO claim
+	// and set the rollout strategy to Recreate.
```

Comment:
how common is this?  is this really the right default for a customer, or would it be better to come up w/ the registry removed and tell the customer to install swift?

Reply:
Cinder was quite common for OpenShift 3.x installs, though of course there the choice between Cinder and Swift was a configuration option.

A lot of customers are not running Swift and for them, the deployment will simply fail right now. This and the self-signed certificates are major blockers for a lot people. They have generally not been receptive to installing Swift so far.

I don't the numbers, but it's not something we thought up. There is a very strong demand for not requiring Swift from our customers.

Reply:
ok